src/Umbraco.Web/Mvc/RenderModelBinder.cs

```diff
@@ -13,8 +12,10 @@ namespace Umbraco.Web.Mvc
     /// <summary>
     /// Allows for Model Binding any IPublishedContent or IRenderModel
     /// </summary>
-	public class RenderModelBinder : DefaultModelBinder, IModelBinder, IModelBinderProvider

```

Comment:
This is a breaking change

Reply:
Removing `IModelBinder` because it's implicitely provided by `DefaultModelBinder` which implements it, so it's redundant. Can put it back, but not breaking.